Documentation/Getting-Started/storage-architecture.md

```diff
@@ -30,3 +30,23 @@ of physical resources, pools, volumes, filesystems, and buckets. Advanced config
 
 Rook is implemented in golang. Ceph is implemented in C++ where the data path is highly optimized. We believe
 this combination offers the best of both worlds.
+
+### Storage Types
+
+#### Volume ReadWriteOnce (RWO)
+
+![Rook Volume ReadWriteOnce Diagram](ceph-storage/Rook%20-%20Storage%20Type%20Volume%20RWO.png)
+
+TODO
```

Comment:
Description paragraphs needed

Reply:
Working on them tomorrow

Reply:
I'm sure I'll have input :), thanks for putting together the draft!